pkg/nvidia-query/nvml/nvml.go

```diff
@@ -583,6 +450,190 @@ func (inst *instance) Get() (*Output, error) {
 	return st, joinedErr
 }
 
+func initAndCheckNVMLSupported(nvmlLib nvml.Interface) (bool, error) {
+	log.Logger.Infow("initializing nvml library")
+	ret := nvmlLib.Init()
+	if ret == nvml.SUCCESS {
+		return true, nil
```

Comment:
don't need to call `nvmlLib.Shutdown()`?

Reply:
oh only used init, we call shutdown outside of this function